murageden/we-connect-v2-react-front-end
# #158820759 Increase Test Coverage

#### What does this PR do?
Increases test coverage
#### Description of Task to be completed?
Increase test coverage
#### How should this be manually tested?
* Setup the application as described on `README`
* Switch to branch `ch-increase-test-coverage-158820759`
* run `npm test`
* Coverage reported should be greater than 20%
#### Any background context you want to provide?
Previously, test coverage was less than 10%
#### What are the relevant pivotal tracker stories?
Pivotal Tracker: https://www.pivotaltracker.com/n/projects/2177751
Story ID: 158820759
#### Questions:
How can this be improved?
